src/main/kotlin/traffic_simulation/Main.kt

```diff
@@ -65,16 +67,28 @@ fun parseInputOfCSV ( fileName : String ): MutableList<Vehicle>{
 
     // insert the parsed information of csv-file in usable lists and use them in functions
     for (record in allRows) {
-
         val id_String : String = record.values[0]
         val wannaDrive_String : String = record.values[1]
 
         val id_Int : Int = id_String.toInt()
         val wannaDrive_Boolean : Boolean = wannaDrive_String.toBoolean()
 
         driveInterest.add ( Vehicle(id = id_Int , wannaDrive = wannaDrive_Boolean ) )
-
     }
-
     return driveInterest
+}
+
+fun parseOutputToCSV (results : MutableList<Vehicle>) {
+    //not sure if this results in writing into the correct file by now
+    FileAccess().getWriter("/" + "results.csv")
```

Comment:
Würde den Dateinamen auch mit in die Attribute von der Funktion nehmen. 

Also für Zeile 81:
`fun parseOutputToCSV (results : MutableList<Vehicle> , fileName: String) {`

dann Zeile 83 entsprechend ändern und nur fileName eintragen. Und weise Zeile 83 noch dem Value writer zu, dass die Zeile so lautet:
`val writer = FileAccess().getWriter(fileName)`